CruCentralCoast/Events/EventsTVC.swift

```diff
@@ -0,0 +1,155 @@
+//
+//  EventsTVC.swift
+//  CruCentralCoast
+//
+//  Created by Cam Stocker on 7/1/18.
+//  Copyright © 2018 Landon Gerrits. All rights reserved.
+//
+
+import UIKit
+
+class EventsTVC: UITableViewController {
+    var selectedCell: EventsTableCell?
+    var originFrame: CGRect?
+    
+    var dataArray = [Event]()
+    
+    var statusBarIsHidden: Bool = false {
+        didSet{
+            UIView.animate(withDuration: 0.25) { () -> Void in
+                self.setNeedsStatusBarAppearanceUpdate()
+            }
+        }
+    }
+    
+    override var prefersStatusBarHidden: Bool {
+        get {
+            return statusBarIsHidden
+        }
+    }
+    
+    override var preferredStatusBarUpdateAnimation: UIStatusBarAnimation {
+        return .slide
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.insertProfileButtonInNavBar()
+        self.tableView.registerCell(EventsTableCell.self)
+        DatabaseManager.instance.getEvents { (events) in
+            self.dataArray = events
+            for event in self.dataArray {
+                UIImage.downloadedFrom(link: event.imageLink, completion: { (image) in
+                    event.image = image
+                    DispatchQueue.main.async {
+                        self.tableView.reloadData()
+                    }
+                })
+            }
+            self.tableView.reloadData()
+        }
+        
+        let longPressGestureRecognizer = UILongPressGestureRecognizer(target: self, action: #selector(handleLongPressGesture(gestureRecognizer:)))
+        longPressGestureRecognizer.minimumPressDuration = 0.1
+        longPressGestureRecognizer.delegate = self
+        self.tableView.addGestureRecognizer(longPressGestureRecognizer)
+    }
+    
+    @objc internal func handleLongPressGesture(gestureRecognizer: UILongPressGestureRecognizer) {
+        let touchPoint = gestureRecognizer.location(in: self.view)
+        if gestureRecognizer.state == .began {
+            if let indexPath = tableView.indexPathForRow(at: touchPoint) {
+                let cell = tableView.cellForRow(at: indexPath)
+                UIView.animate(withDuration: 0.2, animations: {
+                    cell?.transform = CGAffineTransform(scaleX: 0.9, y: 0.9)
+                })
+            }
+        }
+        if gestureRecognizer.state == .ended {
```

Comment:
Should use "else if" or switch statement since the states are mutually exclusive